shallow_backup/__main__.py

```diff
@@ -64,6 +64,10 @@ def cli(show, all, dotfiles, configs, packages, fonts, old_path, new_path, remot
 	# User entered a new path, so update the config
 	if new_path:
 		abs_path = os.path.abspath(new_path)
+		if os.path.isfile(abs_path):
+			print_path_red('New path is an existing file:', abs_path)
+			print_red_bold('Please enter a directory.')
+			sys.exit(1)
```

Comment:
This should be refactored to a function because we need to add the same check in `prompts.path_update_prompt()`